### PR TITLE
CSP-544 all providers endpoint changed from course delivery to Roatp V2

### PR DESCRIPTION
**Pull request by MarkFCain**
…roatp course management

### CODE REVIEW

File: src/Recruit/SFA.DAS.Recruit.Api/Models/GetProviderAddress.cs [hunk 1]
```diff
@@ -15,12 +15,12 @@ public class GetProviderAddress
         {
             return new GetProviderAddress
             {
-                Address1 = source.Address1,
-                Address2 = source.Address2,
-                Address3 = source.Address3,
-                Address4 = source.Address4,
-                Town = source.Town,
-                Postcode = source.Postcode
+                Address1 = source?.AddressLine1,
```
Comment (DevenKShah) [inline src/Recruit/SFA.DAS.Recruit.Api/Models/GetProviderAddress.cs:18]: Can source be null ever? if it can be can we check that first instead of having null check on each assignment?

File: src/Recruit/SFA.DAS.Recruit.sln [hunk 2]
```diff
@@ -42,9 +42,8 @@ Global
 		{D5D4AB58-5E91-42DF-ABFF-E03BCDCD83DB}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{D5D4AB58-5E91-42DF-ABFF-E03BCDCD83DB}.Release|Any CPU.Build.0 = Release|Any CPU
 		{41D340D7-A7E7-4430-B7F3-C5C1A90DD113}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-        {41D340D7-A7E7-4430-B7F3-C5C1A90DD113}.Debug|Any CPU.Build.0 = Debug|Any CPU
-        {41D340D7-A7E7-4430-B7F3-C5C1A90DD113}.Release|Any CPU.ActiveCfg = Release|Any CPU
```
Comment (DevenKShah) [inline src/Recruit/SFA.DAS.Recruit.sln:46]: Is this change required?

**Review by DevenKShah — APPROVED** [review]
Approved with minor code improvement

File: src/Recruit/SFA.DAS.Recruit.Api/Models/GetProviderAddress.cs (hunk 1 above)
Comment (MarkFCain) [reply src/Recruit/SFA.DAS.Recruit.Api/Models/GetProviderAddress.cs:18]: Yes, there are one or two providers out of the hundreds with no address details.  I'll have a look and rework if needed.

File: src/Recruit/SFA.DAS.Recruit.sln (hunk 2 above)
Comment (MarkFCain) [reply src/Recruit/SFA.DAS.Recruit.sln:46]: no, I'll revert.  VS did that, not me...
